core/amber/src/main/resources/application.conf

```diff
@@ -92,8 +92,7 @@ web-server {
 
 fault-tolerance {
     enable-determinant-logging = false
-    # [none, local, hdfs]
-    log-storage-type = "none"
+    log-storage-root-folder-uri = ""
```

Comment:
still better to leave some examples? for instance, how should I use local file system to enable local logging?

Reply:
done